app/helpers/federation_helper.rb

```diff
@@ -94,4 +92,36 @@ def request_portals_names
       content_tag(:span, federated_portal_name(name), style: color ? "color: #{color}" :  "", class: color ? "" : "text-primary")
     end.compact
   end
+
+  def federation_enabled?
+    params[:portals]
+  end
+
+  def federation_error?(response)
+    !response[:errors].blank?
+  end
+
+  def federation_error(response)
+    federation_errors = response[:errors].map{|e| ontology_portal_name(e.split(' ').last.gsub('search', ''))}
```

Comment:
@Bilelkihal I'm not sure of this change, how can I test this?

Reply:
I changed it here https://github.com/ontoportal-lirmm/bioportal_web_ui/pull/739/commits/b57e7c2c12bbefd4476364fcb5dbc0120f6ec4e2